### PR TITLE
SONAR: Remove this useless assignment to local variable

### PR DESCRIPTION
**Pull request by slawekk87**
## Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [ ] My code is written according to the guidelines found [here] (https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/how-to-create-php-files).
- [ ] I have signed the Contributor Licence Agreement between me and YetiForce.

### CODE REVIEW

File: modules/OSSPasswords/actions/Save.php [hunk 1]
```diff
@@ -62,13 +62,13 @@ public function saveRecord(\App\Request $request)
 			// after save we check if encryption is active
 			if ($config) {
 				$sql = 'UPDATE `vtiger_osspasswords` SET `password` = AES_ENCRYPT(?,?) WHERE `osspasswordsid` = ?;';
-				$result = $adb->pquery($sql, [$properPassword, $config['key'], $recordModel->getId()], true);
+				$adb->pquery($sql, [$properPassword, $config['key'], $recordModel->getId()], true);
```
Comment (rskrzypczak) [inline modules/OSSPasswords/actions/Save.php]: query

File: modules/OSSPasswords/actions/Save.php [hunk 2]
```diff
@@ -62,13 +62,13 @@ public function saveRecord(\App\Request $request)
...
 			}
 		} else {
 			$recordModel->save();
 			if ($config) { // when encryption is on
 				$sql = 'UPDATE `vtiger_osspasswords` SET `password` = AES_ENCRYPT(`password`, ?) WHERE `osspasswordsid` = ?;';
-				$result = $adb->pquery($sql, [$config['key'], $recordModel->getId()], true);
```
Comment (rskrzypczak) [inline modules/OSSPasswords/actions/Save.php]: query

File: modules/OSSPasswords/actions/SaveAjax.php [hunk 3]
```diff
@@ -58,15 +58,14 @@ public function process(\App\Request $request)
 		if ($isPassword && $config) {
 			\App\Log::trace('Encrypt new password: ' . $properPassword);
 			$sql = 'UPDATE `vtiger_osspasswords` SET `password` = AES_ENCRYPT(?, ?) WHERE `osspasswordsid` = ?;';
-			$result = $db->pquery($sql, [$properPassword, $config['key'], $recordId], true);
```
Comment (rskrzypczak) [inline modules/OSSPasswords/actions/SaveAjax.php]: query

File: modules/OSSPasswords/actions/SaveAjax.php [hunk 4]
```diff
@@ -58,15 +58,14 @@ public function process(\App\Request $request)
...
-		}
-		// encrypt password added thrue related module
+			$db->pquery($sql, [$properPassword, $config['key'], $recordId], true);
+		} // encrypt password added thrue related module
 		elseif ($isRelatedPassword && $config) {
 			$record = $recordModel->getId();
 			$properPassword = $request->get('password');
 			\App\Log::trace('Encrypt new related module password: ' . $properPassword);
 			$sql = 'UPDATE `vtiger_osspasswords` SET `password` = AES_ENCRYPT(?, ?) WHERE `osspasswordsid` = ?;';
-			$result = $db->pquery($sql, [$properPassword, $config['key'], $record], true);
```
Comment (rskrzypczak) [inline modules/OSSPasswords/actions/SaveAjax.php]: query

File: modules/Settings/OSSPasswords/views/ConfigurePass.php [hunk 5]
```diff
@@ -110,7 +110,7 @@ public function process(\App\Request $request)
 
 					// now encrypt all osspasswords with given key
 					$sql = 'UPDATE `vtiger_osspasswords` SET `password` = AES_ENCRYPT( `password`, ? );';
-					$result = $adb->pquery($sql, [$newPassword], true);
```
Comment (rskrzypczak) [inline modules/Settings/OSSPasswords/views/ConfigurePass.php]: query

File: modules/Accounts/Accounts.php [hunk 6]
```diff
@@ -143,13 +143,8 @@ public function getAccountHierarchy($id, $listColumns = false)
 		// Get the accounts hierarchy (list of child accounts) based on the current account
 		$accountsList[$baseId] = $this->__getChildAccounts($baseId, $accountsList[$baseId], $accountsList[$baseId]['depth']);
 
-		// Create array of all the accounts in the hierarchy
-		$accountHierarchy = $this->getHierarchyData($id, $accountsList[$baseId], $baseId, $listViewEntries);
-
-		$accountHierarchy = ['header' => $listViewHeader, 'entries' => $listViewEntries];
+		return ['header' => $listViewHeader, 'entries' => $listViewEntries];
 		\App\Log::trace('Exiting getAccountHierarchy method ...');
```
Comment (mariuszkrzaczkowski) [inline modules/Accounts/Accounts.php]: ?

File: vtlib/Vtiger/PackageUpdate.php [hunk 7]
```diff
@@ -371,9 +371,7 @@ public function updateCustomViews($modulenode, $moduleInstance)
 		}
 		foreach ($modulenode->customviews->customview as $customviewnode) {
 			$filterInstance = Filter::getInstance($customviewnode->viewname, $moduleInstance->id);
-			if (!$filterInstance) {
-				$filterInstance = $this->importCustomView($modulenode, $moduleInstance, $customviewnode);
```
Comment (mariuszkrzaczkowski) [inline vtlib/Vtiger/PackageUpdate.php]: ?

**Pull request merged**